website/content/docs/api-gateway/configuration/gatewayclassconfig.mdx

```diff
@@ -39,6 +39,7 @@ The following outline shows how to format the configurations in the `GatewayClas
   * [`envoy`](#image-envoy): string | optional
 * [`logLevel`](#loglevel): string | optional
 * [`nodeSelector`](#nodeselector): string | optional
+* [`tolerations`](#tolerations): array of objects | optional
 * [`serviceType`](#servicetype): string | optional
 * [`useHostPorts`](#usehostports): boolean | optional
```

Comment:
I think it's neat having everything in alphabetical order, would like to maintain that here and move the section into the same order below

```suggestion
* [`nodeSelector`](#nodeselector): string | optional
* [`serviceType`](#servicetype): string | optional
* [`tolerations`](#tolerations): array of objects | optional
* [`useHostPorts`](#usehostports): boolean | optional
```